broker-core/src/main/java/io/zeebe/broker/workflow/state/NextValueManager.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Zeebe Broker Core
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package io.zeebe.broker.workflow.state;
+
+import io.zeebe.logstreams.state.StateController;
+import java.nio.ByteOrder;
+import org.agrona.MutableDirectBuffer;
+import org.agrona.concurrent.UnsafeBuffer;
+import org.rocksdb.ColumnFamilyHandle;
+
+public class NextValueManager {
```

Comment:
This class does very similar things that `KeyStateController` does. Can we consolidate them?

Reply:
As discussed I also created for this an issue https://github.com/zeebe-io/zeebe/issues/1387 . Will solve this with #1195